build/build.fsproj

```diff
@@ -4,10 +4,12 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>net7.0</TargetFramework>
     <IsPackable>false</IsPackable>
+    <!-- <NoWarn>NU1904</NoWarn>-->
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
there's a new vulnerability that breaks the build
I see you have 1903 in the build props - if you fix that, I'l rebase off that as its obv not ideal to have this in the PR